tmuxinator.md

```diff
@@ -0,0 +1,200 @@
+# tmuxinator
+###Executing commands sequentially in different panes and windows
+
+Open terminal
+
+$ sudo apt-get install tmux
+
+$ sudo apt-get install ruby
+
+$ sudo su
+
+$ gem install tmuxinator
```

Comment:
If you are changing to root just to install the tmuxinator gem with root priviledges, you can also do `sudo -E gem install tmuxinator`